test/integration/controller/core/workload_controller_test.go

```diff
@@ -105,4 +107,150 @@ var _ = ginkgo.Describe("Workload controller", func() {
 			gomega.Expect(updatedQueueWorkload.Status.Conditions[0].Message).To(testing.Equal(message))
 		})
 	})
+
+	ginkgo.When("Workload with RuntimeClass defined", func() {
+		ginkgo.BeforeEach(func() {
+			resources := corev1.ResourceList{
+				corev1.ResourceCPU: resource.MustParse("1"),
+			}
+			runtimeClass = testing.MakeRuntimeClass("kata", "bar-handler").PodOverhead(resources).Obj()
+			gomega.Expect(k8sClient.Create(ctx, runtimeClass)).To(gomega.Succeed())
+			clusterQueue = testing.MakeClusterQueue("clusterqueue").
+				Resource(testing.MakeResource(corev1.ResourceCPU).
+					Flavor(testing.MakeFlavor(flavorOnDemand, "5").Max("10").Obj()).Obj()).
+				Obj()
+			gomega.Expect(k8sClient.Create(ctx, clusterQueue)).To(gomega.Succeed())
+			queue = testing.MakeQueue("queue", ns.Name).ClusterQueue(clusterQueue.Name).Obj()
+			gomega.Expect(k8sClient.Create(ctx, queue)).To(gomega.Succeed())
+		})
+		ginkgo.AfterEach(func() {
+			gomega.Expect(framework.DeleteRuntimeClass(ctx, k8sClient, runtimeClass)).To(gomega.Succeed())
+			gomega.Expect(framework.DeleteWorkload(ctx, k8sClient, wl)).To(gomega.Succeed())
+			gomega.Expect(framework.DeleteQueue(ctx, k8sClient, queue)).To(gomega.Succeed())
+			gomega.Expect(framework.DeleteClusterQueue(ctx, k8sClient, clusterQueue)).To(gomega.Succeed())
+		})
+
+		ginkgo.It("Should accumulate RuntimeClass's overhead", func() {
+			ginkgo.By("Create workload")
+			wl = testing.MakeWorkload("one", ns.Name).
```

Comment:
Make you can create the workload using `JustBeforeEach` https://onsi.github.io/ginkgo/#separating-creation-and-configuration-justbeforeeach

Reply:
New skill got, thanks for sharing. But considering workloads' configurations in this node are quite different, especially with the former ones, I just don't want to introduce more complexities.